assignment6.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python3
+from statistics import median
+import unittest
+import numpy as np
+
+class RearrangingArray():
+
+	def __init__(self, arrangement, verbose=False):
+
+		self.arrangement = arrangement
+		self.swap_idx = None
+		self.order_dict = None
+		self.verbose = verbose
+
+	def _swap(self, i, j, k):
+		""" Swaps elements on indices i and j using an element on index k """
+		if self.verbose:
+			print(i, k)
+			print(i, j)
+			print(j, k)
+		self.arrangement[i],self.arrangement[k] = self.arrangement[k],self.arrangement[i]
+		self.arrangement[i],self.arrangement[j] = self.arrangement[j],self.arrangement[i]
+		self.arrangement[j],self.arrangement[k] = self.arrangement[k],self.arrangement[j]
+
+	def _move_swap_el(self, j):
```

Comment:
😁it took me a little bit to get that 'el' comes from 'elements' for me would be easier 'elm' maybe
Just an idea 🙂